vhost-device-console/src/backend.rs

```diff
@@ -104,7 +117,7 @@ pub fn start_backend_server(
         vu_console_backend
             .write()
             .unwrap()
-            .assign_input_method(tcp_addr.clone())
+            .assign_input_method(vm_sock.clone())
```

Comment:
Please split the `tcp_addr -> vm_sock` renaming to the refactoring patch suggested before.